packages/paypal-js/types/components/buttons.d.ts

```diff
@@ -201,122 +201,155 @@ type OnShippingAddressChangeActions = {
 
 export type DisplayOnlyOptions = "vaultable";
 
+export type PayPalButtonCreateOrder = (
```

Comment:
Was there any discussion around why these types live in `paypal-js` instead of alongside the components they describe? I'm sure there's some history there, and it's not a blocker on this PR.

Reply:
I don't know about any discussion around this.  Were you thinking of importing it from checkout-components?